File: Entities/ReRateMood.cs
namespace MindYourMoodWeb.Entities
{
    public class ReRateMood
    {
        public int Id { get; set; }
        public int MoodListId { get; set; }
        public int MoodRating { get; set; }
        public Mood Mood { get; set; }
        public ThoughtRecord ThoughtRecord { get; set; }
    }
}

File: ClientApp/src/app/_model/reRateMood.ts
import { Mood } from "./mood";
import { MoodList } from "./moodList";
import { ThoughtRecord } from './thoughtRecord';

export interface ReRateMood {
  id: number;
  moodList: MoodList;
  moodRating: number;
  mood: Mood;
  thoughtRecord: ThoughtRecord;
}
